Source/Core/DolphinQt/MainWindow.cpp

```diff
@@ -255,6 +251,13 @@ MainWindow::MainWindow(std::unique_ptr<BootParameters> boot_parameters,
 
   NetPlayInit();
 
+#ifdef USE_RETRO_ACHIEVEMENTS
+  // This has to be done before CreateComponents() so it's initialized.
+  AchievementManager::GetInstance()->Init();
+  if (Config::Get(Config::RA_ENABLED) && !Config::Get(Config::RA_API_TOKEN).empty())
```

Comment:
Should we log if the api-token isn't defined?  Is that possible?

Reply:
If there's no API token it means there's no player logged in for achievements, which is the natural initial state. Unsure why we'd want to log that? It's not like everyone is going to be using this.

Reply:
Oh, it just seemed odd that there's both a `RA_ENABLED` and a `RA_API_TOKEN` but I haven't looked at this code much.

Reply:
Yeah, they're independent and they both need to be set.